tools/mo/requirements.txt

```diff
@@ -1,13 +1,5 @@
 numpy>=1.16.6
-tensorflow>=1.15.5,<=2.11.0
-mxnet~=1.2.0; sys_platform == 'win32'
-mxnet>=1.7.0.post2,<=1.9.1; sys_platform != 'win32'
 networkx~=2.5; python_version <= "3.6"
 networkx<=2.8.8; python_version > "3.6"
-protobuf>=3.18.1,<4.0.0
-onnx>=1.8.1,<=1.12
 defusedxml>=0.7.1
-urllib3>=1.26.4
-requests>=2.25.1
-fastjsonschema~=2.15.1
 openvino-telemetry>=2022.1.0
```

Comment:
do we need this filtering logic then 
https://github.com/openvinotoolkit/openvino/blob/6c8f8df5182e082633e262a806cf19baae3bbf3c/tools/mo/setup.py#L44-L55
?

Reply:
we need for extras because they do not have `openvino-telemetry`.

Reply:
agreed offline to remove this code